backend/clickhouse/logs.go

```diff
@@ -148,6 +148,39 @@ func (client *Client) LogsKeys(ctx context.Context, projectID int) ([]*modelInpu
 
 }
 
+func (client *Client) LogsKeyValues(ctx context.Context, projectID int, keyName string) ([]string, error) {
```

Comment:
should there be a limit for how many results are returned?

Reply:
Good call! Added in 266a0f40273c29b8e3517377cd81cd985e5cdca1